lib/timers.js

```diff
@@ -26,11 +26,11 @@ var lists = {};
 // the main function - creates lists on demand and the watchers associated
 // with them.
 function insert(item, msecs) {
+  if (msecs < 0) return;
+
   item._idleStart = Timer.now();
   item._idleTimeout = msecs;
```

Comment:
Yes. In fact, the whole separation of `insert()` from `active()` at this point seems redundant. Just pushed an additional commit.